### PR TITLE
🔧 Modifications in Apply - Build profile for Mentees plus issues fixing

### LINKED ISSUE #939: Mentee application and build profile forms, some changes
There are many fields in the Mentee application and Build profile forms that needs to be adjusted

This excel will have the final list

Requirements
1) Existing data fields entered by users already existing should not be removed, no data loss is accepted
2) All data fields must be exported in the Admin excel sheet downloaded
3) All data fields should be displayed to Admin user before APPROVING an application when they open the application details for Mentee
4) All data fields should be displayed in the user Edit Profile screen after profile is created.

https://docs.google.com/spreadsheets/d/1NgVne9tioX2j3Yt7tJ6YkVSEU4dGPDLq1BLhWlu3t5M/edit?usp=sharing

### PR DESCRIPTION
**Pull request by SolMon9099**
## Description :star2:

Fixes #939

### CODE REVIEW

File: backend/api/models/MenteeApplication.py [hunk 1]
```diff
@@ -9,12 +9,13 @@ class MenteeApplication(Document, Mixin):
 
     email = StringField(required=True)
     name = StringField(required=True)
-    age = StringField(required=True)
+    age = StringField(required=False)
```
Comment (robert-dm) [inline backend/api/models/MenteeApplication.py:12]: @SolMon9099 why it is required false?

File: backend/api/models/MenteeApplication.py [hunk 2]
```diff
@@ -9,12 +9,13 @@ class MenteeApplication(Document, Mixin):
...
     organization = StringField()
     immigrant_status = ListField(StringField(), required=True)
     Country = StringField()
     identify = StringField(required=True)
-    language = StringField(required=True)
+    # language = StringField(required=True)
```
Comment (robert-dm) [inline backend/api/models/MenteeApplication.py:17]: @SolMon9099 please remove comment

File: backend/api/utils/constants.py [hunk 3]
```diff
@@ -39,6 +39,16 @@
...
+# Education Level
+EDUCATION_LEVEL = {
+    "elementary": "Elementary school",
+    "high": "High school",
+    "technical": "Technical school",
+    "bachelor": "Bachelor",
+    "masters": "Master",
+    "doctorate": "Doctorate",
+}
+
```
Comment (robert-dm) [inline backend/api/utils/constants.py:51]: @SolMon9099 can we please add these to the translations.

File: backend/api/utils/request_utils.py [hunk 4]
```diff
@@ -132,7 +132,7 @@ class MentorApplicationForm(Form):
 class MenteeApplicationForm(Form):
     email = StringField(validators=[InputRequired()])
     name = StringField(validators=[InputRequired()])
-    age = StringField(validators=[InputRequired()])
+    # age = StringField(validators=[InputRequired()])
```
Comment (robert-dm) [inline backend/api/utils/request_utils.py:135]: @SolMon9099 pls remove comments

File: backend/api/views/download.py [hunk 5]
```diff
@@ -209,11 +209,13 @@ def download_mentee_apps(apps):
             [
                 acct.name,
                 acct.email,
-                acct.age,
+                # acct.age,
```
Comment (robert-dm) [inline backend/api/views/download.py:212]: @SolMon9099 please remove comment

File: backend/api/views/download.py [hunk 6]
```diff
@@ -225,7 +227,7 @@ def download_mentee_apps(apps):
     columns = [
         " Full Name",
         "email",
-        "age",
+        # "age",
```
Comment (robert-dm) [inline backend/api/views/download.py:230]: @SolMon9099 pls remove comment

File: frontend/src/components/MenteeAppInfo.js [hunk 7]
```diff
@@ -27,13 +27,13 @@ function NewMentorAppInfo({ info }) {
...
-          {info.organization ? info.organization : info.partner}
+          {info.organization ? info.organization : "No affiliation"}
         </div>
       </div>
-      <div className="single-info-section info2">
+      {/* <div className="single-info-section info2">
         <div className="question">{"Age"}</div>
         <div className="answer">{info.age}</div>
-      </div>
+      </div> */}
```
Comment (robert-dm) [inline frontend/src/components/MenteeAppInfo.js:36]: @SolMon9099 pls remove comments

File: frontend/src/components/pages/MenteeApplication.js [hunk 8]
```diff
@@ -256,7 +274,7 @@ function MenteeApplication({ email, role, onSubmitSuccess, onSubmitFailure }) {
               },
             ]}
           />
-        </Form.Item>
+        </Form.Item> */}
```
Comment (robert-dm) [inline frontend/src/components/pages/MenteeApplication.js:277]: @SolMon9099 pls remove comments

File: frontend/src/components/pages/MenteeApplication.js [hunk 9]
```diff
@@ -291,8 +309,18 @@ function MenteeApplication({ email, role, onSubmitSuccess, onSubmitFailure }) {
...
+          label={t("menteeApplication.countryPlaceholder")}
+          name="country"
+          rules={[
+            {
+              required: true,
+              message: t("common.requiredHearAboutUs"),
+            },
+          ]}
+        >
+          {/* <Input placeholder={t("menteeApplication.countryPlaceholder")} /> */}
```
Comment (robert-dm) [inline frontend/src/components/pages/MenteeApplication.js:322]: @SolMon9099 pls remove comments

**Review by robert-dm — CHANGES_REQUESTED** [review]
@SolMon9099 pls check comments

File: backend/api/models/MenteeApplication.py (hunk 1 above)
Comment (SolMon9099) [reply backend/api/models/MenteeApplication.py:12]: I removed age from apply form. So it is necessary to make it False.

File: backend/api/models/MenteeApplication.py (hunk 2 above)
Comment (SolMon9099) [reply backend/api/models/MenteeApplication.py]: Removed

File: backend/api/utils/request_utils.py (hunk 4 above)
Comment (SolMon9099) [reply backend/api/utils/request_utils.py]: removed

File: frontend/src/components/MenteeAppInfo.js (hunk 7 above)
Comment (SolMon9099) [reply frontend/src/components/MenteeAppInfo.js]: removed

File: frontend/src/components/pages/MenteeApplication.js (hunk 8 above)
Comment (SolMon9099) [reply frontend/src/components/pages/MenteeApplication.js]: removed

File: frontend/src/components/pages/MenteeApplication.js (hunk 9 above)
Comment (SolMon9099) [reply frontend/src/components/pages/MenteeApplication.js]: removed

File: backend/api/utils/constants.py (hunk 3 above)
Comment (SolMon9099) [reply backend/api/utils/constants.py:51]: I already added translations. Please check after merge

**Review by robert-dm — APPROVED** [review]

**Pull request merged**